src/lib.rs

```diff
@@ -20,6 +21,7 @@ pub mod cuda;
 #[cfg(feature = "opencl")]
 pub mod opencl;
 
+pub use corecounts::CORE_COUNTS;
```

Comment:
I would call this `CUDA_CORE_COUNTS`, as this is what this really represents. AMD perf would be calculated differentlyf

Reply:
What about `CUDA_CORES` to be shorter?

Reply:
sure